### ISSUE TITLE
BootstrapperBase missing from netcoreapp3.0 target

### ISSUE THREAD

**Issue by WilliamForsdal** [Caliburn-Micro/Caliburn.Micro#632]
Hey,
I have a WPF project targeting netcoreapp3.0, and using Caliburn.Micro 4.0.62-alpha there are some types I cannot find in the assembly; BootstrapperBase, IWindowManager/WindowManager, XamlPlatformProvider, and maybe more.
For example:
```csharp
public class Bootstrapper : BootstrapperBase
{
}
```
gives me error CS0246 missing using/assembly reference etc.

If I switch target framework to net452 the issue disappears. 
Also with target framework netcoreapp3.0 and caliburn micro verison 3.2.0 it works.

**Comment by WilliamForsdal** [comment]
I noticed the latest commits adds a netcoreapp3.0 build target. I guess this has to be made into a release to be able to use it in a netcoreapp3.0?

**Comment by WilliamForsdal** [comment]
I solved the issue by using version 4.0.89-alpha.


For anyone with the same issue, if you want to use the later version, the command specified to install the package from myget.org does not work if the project is using transitive package management. Running the command `Install-Package Caliburn.Micro -Version 4.0.89-alpha -Source https://www.myget.org/F/caliburn-micro-builds/api/v3/index.json` results in an error message: `The 'Source' parameter is not respected for the transitive package management based project(s) YourProject. The enabled sources in your NuGet configuration will be used.`
I fixed this by just adding `https://www.myget.org/F/caliburn-micro-builds/api/v3/index.json` as a nuget package source, and everything works well. (Tools -> Options -> NuGet Package Manager -> Package Sources, add a new source with Source set to the myget url).
